OpenKh.WinShell.IMDThumbnail/IMDThumbnail.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.IO;
+using System.Drawing;
+using System.Runtime.InteropServices;
+
+using SharpShell.Attributes;
+using SharpShell.SharpThumbnailHandler;
+
+using OpenKh.Kh2;
+using OpenKh.Imaging;
+
+namespace OpenKh.WinShell.IMDUtilities
+{
+    [Obsolete]
```

Comment:
Why is this marked as Obsolete?

Reply:
You see, because FileExtension is an obsolete feature but at the same time ClassOfExtension (which is meant to replace FileExtension) does not work with Thumbnail Generation.
This should be addressed in the next version of SharpShell.